gemma-core/pom.xml

```diff
@@ -260,7 +264,7 @@
                                     <sqlCommand>
 										<![CDATA[
                                         DROP DATABASE IF EXISTS ${gemma.testdb.name};
-                                        CREATE DATABASE ${gemma.testdb.name}
+                                        CREATE DATABASE ${gemma.testdb.name} CHARACTER SET utf8mb4;
```

Comment:
We could simply revert this line if we change our default character set to utf8mb4. I think starting with MySQL 8, it is the default.